integration-tests/tests/run-workflow.js

```diff
@@ -28,38 +26,71 @@ const testRunWorkflowFn = _.flow(
-  await click(page, `//*[@role="checkbox" and contains(@aria-label, "${testEntity.name}")]`)
-  await click(page, clickable({ text: 'OK' }))
-  await click(page, clickable({ text: 'Run analysis' }))
+    await Promise.all([
+      page.waitForNavigation(),
+      click(page, clickable({ text: 'Launch' }))
+    ])
 
-  await Promise.all([
-    page.waitForNavigation(),
-    click(page, clickable({ text: 'Launch' }))
-  ])
+    await pRetry(async () => {
+      try {
+        await findInGrid(page, 'Succeeded', { timeout: 65 * 1000 }) // long enough for the submission details to refresh
+      } catch (e) {
+        throw new Error(e)
+      }
+    }, { retries: 10, factor: 1 })
 
-  await pRetry(async () => {
-    try {
-      await findInGrid(page, 'Succeeded', { timeout: 65 * 1000 }) // long enough for the submission details to refresh
-    } catch (e) {
-      throw new Error(e)
+    await click(page, navChild('data'))
+    await click(page, clickable({ textContains: 'test_entity' }))
+    await findInDataTableRow(page, testEntity.name, testEntity.attributes.input)
+  } finally {
+    if (!isWorkspaceSingleUse) {
+      const methodConfigs = await page.evaluate((namespace, name) => {
+        return window.Ajax().Workspaces.workspace(namespace, name).listMethodConfigs()
+      }, billingProject, workspaceName)
+      const methodConfigToDelete = _.find({ methodRepoMethod: { methodName: workflowName } }, methodConfigs)
+      await page.evaluate((workspaceNamespace, workspaceName, methodNamespace, methodName) => {
+        return window.Ajax().Workspaces.workspace(workspaceNamespace, workspaceName).methodConfig(methodNamespace, methodName).delete()
+      }, billingProject, workspaceName, methodConfigToDelete.namespace, methodConfigToDelete.name)
     }
-  }, { retries: 10, factor: 1 })
+  }
+}
 
-  await click(page, navChild('data'))
-  await click(page, clickable({ textContains: 'test_entity' }))
-  await findInDataTableRow(page, testEntity.name, testEntity.attributes.input)
+const testRunWorkflowFn = _.flow(
+  withWorkspace,
+  withUserToken
+)(async ({ billingProject, page, testUrl, token, workflowName, workspaceName }) => {
+  await testRunWorkflowHelper(billingProject, page, testUrl, token, workflowName, workspaceName)
+})
+
+const testRunWorkflowWithV1WorkspaceFn = _.flow(
+  withV1Workspace,
+  withRegisteredUser
+)(async ({ billingProject, page, testUrl, token, workflowName, v1WorkspaceName }) => {
+  await testRunWorkflowHelper(billingProject, page, testUrl, token, workflowName, v1WorkspaceName, false)
 })
 
+
 const testRunWorkflow = {
   name: 'run-workflow',
   fn: testRunWorkflowFn,
   timeout: 15 * 60 * 1000
 }
 
-module.exports = { testRunWorkflow }
+const testRunWorkflowWithV1Workspace = {
+  name: 'run-workflow-v1-workspace',
+  fn: testRunWorkflowWithV1WorkspaceFn,
+  timeout: 15 * 60 * 1000
+}
+
+module.exports = { testRunWorkflow, testRunWorkflowWithV1Workspace }
```

Comment:
Same in this file, if one of these tests is intended to be around forever and one is only intended to be around until some external condition is met (i.e., PPW migration is done) then let's comment as such and leave a marker for how to find this and how to identify when it can be safely removed.

Reply:
Filed https://broadworkbench.atlassian.net/browse/CA-1366